makefile

```diff
@@ -11,6 +11,8 @@ packages += rest-client rest-data rest-route rest-model migrations trigger model
 lintOnlyPackages := testutil model-manifest
 orgPath := github.com/evergreen-ci
 projectPath := $(orgPath)/$(name)
+lobsterTempDir := $(shell pwd)/$(buildDir)/lobster-temp
```

Comment:
If you put this below the EVGHOME definition you can just do:
`lobsterTempDir := $(EVGHOME)/$(buildDir)/lobster-temp).